internal/server/data/migrations.go

```diff
@@ -558,3 +566,85 @@ func dropOrganizationNameIndex() *migrator.Migration {
 		},
 	}
 }
+
+func addRLSUser() *migrator.Migration {
+	return &migrator.Migration{
+		ID: "2022-08-15T15:19",
+		Migrate: func(tx migrator.DB) error {
+			logging.Infof("adding RLS user")
+			var val int
+			err := tx.QueryRow("SELECT 1 FROM pg_roles WHERE rolname = 'infra_user'").Scan(&val)
```

Comment:
Should we be messing with database privileges here? That feels unexpected in a migration, I'd rather move the process to something manual until we have a workflow to handle it through Infra personally.